test/commands/UserInputUtil.test.ts

```diff
@@ -567,6 +567,14 @@ describe('UserInputUtil', () => {
                 placeHolder: 'Choose the smart contract package that you want to delete'
             });
         });
+
+        it('should show error and return when there are no open packages', async () => {
+            mySandBox.stub(PackageRegistry.instance(), 'getAll').resolves([]);
```

Comment:
You don't need to stub the getAll you should just call the clear function on the registry

Reply:
Thanks!